portability-spi-transfer/src/main/java/org/dataportabilityproject/spi/transfer/provider/TransferServiceProviderRegistry.java

```diff
@@ -11,5 +15,8 @@
      * @param serviceId the service id
      */
     TransferServiceProvider getServiceProvider(String serviceId);
+    Exporter<AuthData, DataModel> getExporter(String serviceId, PortableType transferDataType);
```

Comment:
I think TransferServiceProvider already has getExporter and getImporter, can we grab them from there?

Reply:
As discussed, I removed the method getServiceProvider  and left these two. We can add it back if we find that we need it